cpu/esp32/Makefile.include

```diff
@@ -122,6 +122,10 @@ ifeq ($(QEMU), 1)
     CFLAGS += -DQEMU
 endif
 
+ifneq (,$(filter lwip,$(USEMODULE)))
+    CFLAGS += -DTCPIP_THREAD_PRIO=2
```

Comment:
What is this about?

Reply:
If it is required, then this shouldn't be hidden in the platform code.

Reply:
It is platform specific. By default the [lwIP's `tcpip` thread](https://www.nongnu.org/lwip/2_0_x/group__lwip__opts__thread.html#ga42b2c7a3042d7c3efd00f367f5837435) has a priority of 1. However, in ESP32, the thread that handles hardware interrupts from the WiFi interface has also a priority of 1. From my point of view, the hardware driver should have a higher priority than the TCP/IP thread to be sure that hardware is always handled in time.

I figured it out and changed it when I was investigating the problem with the duplicate ACKs. This change was already covered by the test by @MichelRottleuthner.

Reply:
I have no doubts that it works, I just fear that it might cause problems later on, with people not expecting that something related to an external package is configured within a `cpu` Makefile. How about flipping it around and putting it in `pkg/lwip/Makefile.include`? Including some explanation why it is required.